README.md

```diff
@@ -1,55 +1,129 @@
 # VirtoCommerce.CatalogPersonalization
-VirtoCommerce.CatalogPersonalization module adds personalization feature to the products catalog so that store owner now could define different user experiences for one catalog categories and products.
 
-### Key features:
-* control the visibility of catalog objects such as Product and Categories, through manual tagging of these objects with special property `Tags` which can contain multiple predefined values which also may be defined for customer profile
-* inheritance `Tags` between catalogs objects depend on catalog taxonomy (categories inherit this property from parent catalog or category, products inherit this property from parent category)
-* allow using `User groups` from customer profile to products or categories filtration in the storefront search
-* the two different policies for `Tags` propagation in catalog objects hierarchy 
-    - `UpTree` propagate the tags from descendants to parents up the hierarchy
-    ![image](https://user-images.githubusercontent.com/7566324/62931481-ba630c00-bdbe-11e9-9cdf-6d05e955721b.png)
-    
-    
-    - `DownTree` inherites the tags by descendants from their parrent down the hierarchy
-    ![image](https://user-images.githubusercontent.com/7566324/62931421-a3241e80-bdbe-11e9-8f02-fd22d0fbcc6f.png)
+The main concept of VC Personalization module is to personalize the issuing of catalog, price lists and marketing actions using Tags and User Groups.
```

Comment:
Maybe rephrase, I do not clearly understand the purpose of the module from this sentense